modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -0,0 +1,173 @@
+kind: Secret
+metadata:
+  name: kni1-master-0-bmc-secret
+  namespace: openshift-machine-api
+data:
+  password: <username>
+  username: <password>
+type: Opaque
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: kni1-master-0
+  namespace: openshift-machine-api
+spec:
+  automatedCleaningMode: disabled
+  bmc:
+    address: redfish-virtualmedia+http://192.168.124.113:8000/redfish/v1/Systems/f87eaf82-b32d-4291-ace6-b28677964e78
+    credentialsName: kni1-master-0-bmc-secret
+  bootMACAddress: aa:aa:aa:aa:ab:03
+  bootMode: UEFI
+  externallyProvisioned: false
+  hardwareProfile: unknown
+  online: true
+EOF
+----
++
+After the inspection is complete, the `baremetalhost` object is created and available to be provisioned.
+
+. View available `baremetalhost` objects:
++
+[source,terminal]
+----
+$ oc get bmh -n openshift-machine-api
+----
++
+.Example output
+[source,terminal]
+----
+NAME           STATE                    CONSUMER                   ONLINE   ERROR   AGE
+kni1-master-0  available                ocp-hkw9p-master-0         true             1h10m
+kni1-master-1  externally provisioned   ocp-hkw9p-master-1         true             4h53m
+kni1-master-2  externally provisioned   ocp-hkw9p-master-2         true             4h53m
+kni1-worker-0  provisioned              ocp-hkw9p-worker-0-ktmmx   true             4h53m
+kni1-worker-1  provisioned              ocp-hkw9p-worker-0-l2zmb   true             4h53m
+----
++
+There are no `MachineSet` objects for control plane nodes, so you must create a `MachineSet` object instead. You can copy the image URL and checksum from the other `BareMetalHost` objects.
+
+. Create a `MachineSet` object:
++
+[source,terminal]
+----
+$ cat <<EOF | oc apply -f -
+apiVersion: machine.openshift.io/v1beta1
+kind: Machine
+metadata:
+  annotations:
+    metal3.io/BareMetalHost: openshift-machine-api/kni1-master-0
+  labels:
+    machine.openshift.io/cluster-api-cluster: kni1
+    machine.openshift.io/cluster-api-machine-role: master
+    machine.openshift.io/cluster-api-machine-type: master
+  name: kni1-master-0
+  namespace: openshift-machine-api
+spec:
+  metadata: {}
+  providerSpec:
+    value:
+      apiVersion: baremetal.cluster.k8s.io/v1alpha1
+      customDeploy:
+        method: install_coreos
+      hostSelector: {}
+      image:
+        checksum: ""
+        url: ""
+      kind: BareMetalMachineProviderSpec
+      metadata:
+        creationTimestamp: null
+      userData:
+        name: master-user-data-managed
+EOF
+----
++
+. To define and create the `BareMetalHost`, `Secret`, and `MachineSet` objects in a single step, create a YAML file (`example.yaml`) with their definitions and run the following command:
++
+[source,terminal]
+----
+$ oc create -f example.yaml
+----
++
+The provisioning process uses the baremetal-operator to install RHCOS and prepare the host to be added to the cluster. 
++
+. To view the `baremetalhost` objects, run the following command:
```

Comment:
```suggestion
. To view the `BareMetalHost` objects, run the following command:
```
For consistency as it is using `BareMetalHost` in all the other sentences

Reply:
Updated as suggested.